src/nerpa_pipeline/handle_rban.py

```diff
@@ -101,6 +155,9 @@ def putative_backbones(G, min_nodes=2):
         except:
             pass
 
+        '''
+        try to parse the component as a chupa-choops graph (a path attached to a simple cycle)
```

Comment:
I think `a "lollipop" graph` (with quotes) is better than `a chupa-choops graph`